test/ngResource/resourceSpec.js

```diff
@@ -29,6 +29,9 @@ describe('basic usage', function() {
         }
       }
 
+    },
+    {
+      cancellable: true
```

Comment:
Nit: It probably doesn't matter, but I would rather keep changes local, instead of affecting all tests. I.e. create a new, cancellable resource for your test.